corehq/apps/reports/v2/filters/case_report.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from django.utils.translation import ugettext_lazy
+
+from corehq.apps.reports.standard.cases.utils import (
+    query_all_project_data,
+    query_deactivated_data,
+    get_case_owners,
+)
+from corehq.apps.reports.v2.endpoints.case_owner import CaseOwnerEndpoint
+from corehq.apps.reports.v2.models import BaseReportFilter
+from corehq.apps.reports.filters.case_list import CaseListFilter as EMWF
+
+
+class Widget(object):
+    SELECT2_MULTIPLE = 'select2_multiple'
+
+
+class CaseOwnerReportFilter(BaseReportFilter):
+    title = ugettext_lazy("Case Owner(s)")
+    name = 'report_case_owner'
+    endpoint_slug = CaseOwnerEndpoint.slug
+    widget = Widget.SELECT2_MULTIPLE
```

Comment:
Is this for future work? I don't understand what the `widget` attribute does or how the string `"select2_multiple"` fits in.